plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/status/StatusUtil.java

```diff
@@ -32,4 +45,12 @@ public static IStatus error(Object origin, String message, Throwable error) {
     }
   }
 
+  private static IStatus errorInternal(String bundleOrClassname, String message, Throwable error) {
```

Comment:
I could not find a better name for this method which is basically where all the public method calls end up. It is intended to be used only by this class. Anyway, I remove  `Internal` from the name.